udata/core/reuse/views.py

```diff
@@ -97,11 +99,37 @@ def get_context(self):
         context.update(
             followers=followers,
             can_edit=ReuseEditPermission(self.reuse),
-            discussions=Discussion.objects(subject=self.reuse)
+            discussions=Discussion.objects(subject=self.reuse),
         )
 
         return context
 
+    def get_json_ld(self):
+        reuse = self.reuse
+        result = {'@context': "http://schema.org",
+                  '@type': "CreativeWork",
+                  "alternateName": reuse.slug,
+                  "dateCreated": reuse.created_at.isoformat(),
+                  "dateModified": reuse.last_modified.isoformat(),
+                  "url": url_for('reuses.show', reuse=reuse, _external=True),
+                  "name": reuse.title,
+                  "description": reuse.description,
+                  "isBasedOnUrl": reuse.url
+            }
+
+        if reuse.organization:
```

Comment:
Same as above: Models should have the JSON-LD serialization, and you can directly affect to `result['author']`